src/Compilers/VisualBasic/Portable/Scanner/ScannerInterpolatedString.vb

```diff
@@ -140,17 +138,16 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
             Dim offset = 0
             Dim pendingWhitespace = 0
             Dim valueBuilder = GetScratch()
-
-            Do While CanGet(offset)
-
-                Dim c = Peek(offset)
+            Dim c, c1 As Char
+            Do While TryPeek(offset, c)
 
                 ' Any combination of fullwidth and ASCII curly braces of the same direction is an escaping sequence for the corresponding ASCII curly brace.
                 ' We insert that curly brace doubled and because this is the escaping sequence understood by String.Format, that will be replaced by a single brace.
                 ' This is deliberate design and it aligns with existing rules for double quote escaping in strings.
+                Dim ok = TryPeek(offset+1, c1)
```

Comment:
Consider inlining TryPeek(offset + 1, c1), otherwise we might call it in cases when the call is not needed.